yb-voyager/src/srcdb/oracle.go

```diff
@@ -34,6 +34,18 @@ func (ora *Oracle) Connect() error {
 	return err
 }
 
+func (ora *Oracle) Disconnect() {
+	if ora.db == nil {
+		log.Infof("No connection to the source database to close")
+		return
+	}
+
+	err := ora.db.Close()
+	if err != nil {
+		utils.PrintAndLog("Failed to close connection to the source database: %s", err)
```

Comment:
hmm, err != nil can be if there is still running something running from the previous steps, and it could not close the connection.
In that case i was thinking to print the message to screen, to bring it to user's notice. Although it will happen rarely i guess, ideally the connection should get closed easily.

Reply:
Given that there's nothing the user can really do about that information(ok, in this voyager run, the connection was not closed properly, now what?), I think it should be okay to keep it in the log.

Reply:
ok, going with just logging it